src/core/lib/iomgr/ev_posix.h

```diff
@@ -84,10 +86,16 @@ void grpc_event_engine_shutdown(void);
 /* Return the name of the poll strategy */
 const char* grpc_get_poll_strategy_name();
 
+/* Returns true if polling engine can track errors separately, false otherwise
```

Comment:
Need a more detailed comment here. We should explain what "tracking errors" means..

Reply:
You are right. Someone without context reading this will not understand at all.